packages/mobile/src/apollo/__mockData__.ts

```diff
@@ -1,93 +1,81 @@
-import {
-  EventTypeNames,
-  HomeExchangeFragment,
-  HomeTransferFragment,
-  UserTransactionsData,
-} from 'src/apollo/types'
+import { HomeExchangeFragment, HomeTransferFragment, UserTransactionsData } from 'src/apollo/types'
+import { CURRENCY_ENUM } from 'src/geth/consts'
 import { SENTINEL_INVITE_COMMENT } from 'src/invite/actions'
-import { TransactionTypes } from 'src/transactions/reducer'
+import { TransactionTypes, TransferTransactionTypes } from 'src/transactions/reducer'
 
 export const invitedAddress = '0x1b173'
 
 const exchangeDollar: HomeExchangeFragment = {
-  __typename: EventTypeNames.Exchange,
-  type: 'EXCHANGE' as TransactionTypes,
+  type: 'EXCHANGE' as TransactionTypes.EXCHANGE,
```

Comment:
Why not just use the enum's actual value? (not sure who did this originally)

Reply:
I think this is a more realistic mock of what would be returned?

Reply:
I think Connors point is that `TransactionTypes.Exchange` === 'EXCHANGE' so why hardcode the value here?
And I think the reason it was originally hard coded in the first place is that the auto-gen typing tool did it this way